src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin.csproj

```diff
@@ -40,11 +39,12 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.AzureStack.Management.Admin.Compute, Version=0.1.0-preview">
-      <HintPath>..\..\..\packages\Microsoft.AzureStack.Management.Compute.Admin.0.1.0-preview\lib\net452\Microsoft.AzureStack.Management.Compute.Admin.dll</HintPath>
+    <Reference Include="Microsoft.AzureStack.Management.Admin.Compute, Version=0.3.0-preview">
+      <HintPath>..\..\..\packages\Microsoft.AzureStack.Management.Compute.Admin.0.3.0-preview\lib\net452\Microsoft.AzureStack.Management.Compute.Admin.dll</HintPath>
     </Reference>
   </ItemGroup>
   <ItemGroup>
+    <None Include="Azs.Compute.Admin\Generated.PowerShell.Commands\SwaggerDefinitionCommands\New-QuotaCustomObject.ps1" />
```

Comment:
Revert this change.

Reply:
r u suggesting not to use New-QuotaCustomObject itself ?

Reply:
No, if you leave this then the files will be copied into the directory containing the DLLs.  There are two purposes of the csproj,

1.  Extract Nuget files into Azs.Compute.Admin\ref\fullclr
2.  Copy everything to src\Stack\...\<Output>

Reply:
removed.